owslib/wmts.py

```diff
@@ -136,7 +136,7 @@ def __getitem__(self, name):
             raise KeyError("No content named %s" % name)
 
     def __init__(self, url, version='1.0.0', xml=None, username=None, password=None,
-                 parse_remote_metadata=False, vendor_kwargs=None, headers=None, auth=None,
+                 parse_remote_metadata=False, vendor_kwargs=None, headers=None, cookies=None, auth=None,
```

Comment:
It would be safer to add the new cookies parameter at the end of the list to avoid breaking existing codebases